mne/viz/topomap.py

```diff
@@ -568,7 +568,10 @@ def _get_extra_points(pos, extrapolate, origin, radii):
     else:
         assert extrapolate == 'head'
         # return points on the head circle
-        angle = np.arcsin(distance / 2 / np.mean(radii))
+        angle = np.arcsin(distance / np.mean(radii))
+        n_pnts = np.round(2 * np.pi / angle)
+        angle = 2 * np.pi / n_pnts
+
         points_l = np.arange(0, 2 * np.pi, angle)
```

Comment:
At this point I would just do `np.linspace(0, 2 * np.pi, n_pts, endpoint=False)` -- it ensures you get exactly `n_pts` which you are not guaranteed from the `np.arange` operating on `float` due to precision issues, and you don't need to recompute `angle`.